core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -483,8 +483,6 @@ private static GetConfigurationPResponse loadConfiguration(InetSocketAddress add
       LOG.info("Alluxio client has loaded configuration from meta master {}", address);
       return response;
     } catch (io.grpc.StatusRuntimeException e) {
-      AlluxioStatusException ase = AlluxioStatusException.fromStatusRuntimeException(e);
-      LOG.warn("Failed to handshake with master {} : {}", address, ase.getMessage());
```

Comment:
Do we need `ase.getMessage()` anywhere?

Reply:
it isn't lost, we pass `e` as the cause to the thrown exception

Reply:
actually to be safe, I'm adding it to the root message as well

Reply:
o, I guess what I was wondering i was wondering was before, we did this `ase = AlluxioStatusException.fromStatusRuntimeException(e)`. Does that add any useful information, since we don't do that translation any more?

Reply:
it doesn't add any useful information, it just converts between types